config/example/node.yaml

```diff
@@ -10,6 +10,8 @@ pprof:
   enabled: true
   address: localhost:6060  # endpoint for Node profiling
   shutdown_timeout: 15s  # timeout for profiling HTTP server graceful shutdown
+  enable_block: true # toggles goroutine block profiling. Disabled by default. May be updated in runtime.
```

Comment:
a few words about why it is optional? these are the most expensive profiles

Reply:
done